crates/sui-adapter-transactional-tests/tests/dev_inspect/reference_return_value.move

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// attempt to return a reference from a function
+
+//# init --addresses test=0x0 --accounts A
+
+//# publish
+
+module test::m {
+    public fun return_ref(n: &u64): &u64 {
+        n
+    }
+}
+
+//# programmable --sender A --inputs 0 --dev-inspect
+//> 0: test::m::return_ref(Input(0));
```

Comment:
```suggestion
//> 0: test::m::return_ref(Input(0));
```

Reply:
Nit missing newline (lots of folks' editors will add this leading to spurious changes)